ui/src/actions/remoteAccess.js

```diff
@@ -86,9 +86,9 @@ export function logoutUser(username) {
   };
 }
 
-export function respondToControlRequest(giveControl = true, message = '') {
+export function respondToControlRequest(giveCtrl = true, message = '') {
   return async (dispatch) => {
-    await sendRespondToControlRequest(giveControl, message);
+    await sendRespondToControlRequest(giveCtrl, message);
```

Comment:
Granted, it's not really more readable here... There's an action creator function called `giveControl`. I would have renamed it to `accept` or something, but the API expects a `giveControl` parameter, so I chose to keep the naming as close as possible.